docs/source/user/10minute_intro.md

```diff
@@ -1,7 +1,246 @@
 # 10 Minutes to SeismoStats 
 
-This tutorial covers the most essential usage of the SeismoStats package.
+Welcome to a quick overview of **SeismoStats** — a Python package which is designed to simplify the analysis of seismic catalogs and to provide a foundation for more advanced seismicity studies.
+
+This overview introduces the core features of SeismoStats, with a focus on three primary analysis goals:
+
+- **b-value** calculation  
+- **a-value** estimation  
+- **Magnitude of completeness** estimations  
+
+Additionally, we demonstrate how to easily visualize your data and explore key features of your catalog.
+
+---
+
+A central feature of SeismoStats is the `Catalog` object, which offers a quick and flexible way to get started. It is built on top of a `pandas.DataFrame`, meaning all standard pandas methods are available and fully supported. Beyond the basic event data (e.g., magnitude, time), the `Catalog` object also includes:
+
+- *Attributes*: User-defined properties such as `delta_m` and catalog properties, which are estimated by the methods (e.g. `b_value`, and `a_value`).
+
+- *Methods*: Tools for visualization, statistical analysis, and data conversion.
+Importantly, many methods both **use** and **update** the catalog's properties. This architecture is illustrated below:
+
+![catalog_class](../_static/catalog_class.png "Overview on catalog class")
+
+## 1 Creating a Catalog
+
+Catalogs can be created from:
+
+- `pandas.DataFrame`  
+- Python dictionaries
+- Existing catalogs in **QuakeML** or **OpenQuake** formats  
+
+You can also fetch earthquake data directly from FDSN servers such as **EIDA** and **USGS** using the built-in FDSN client.
+
+> **Note:** Your catalog must include a `magnitude` column. For full functionality (especially plotting and analysis), it is recommended to also include `time`, `latitude`, `longitude`, and `depth`.
+
+
+### 1.1 Example: Creating a Catalog from a DataFrame
+
+```python
+>>> from seismostats import Catalog
+>>> import pandas as pd
+
+>>> # Create a DataFrame with earthquake data
+>>> df = pd.DataFrame({
```

Comment:
maybe also show, that you can instantiate the Catalog also directly like this (`Catalog({...})`).